fs/fs.go

```diff
@@ -483,6 +483,17 @@ func getDiskStatsMap(diskStatsFile string) (map[string]DiskStats, error) {
 				return nil, error
 			}
 		}
+
+		major64, error :=  strconv.ParseUint(words[0], 10, 64)
```

Comment:
Use `err` instead of `error` as error-storing variable, please.

Reply:
Sure, I have renamed to "err" in my latest commit.